tests/unit/test_client.py

```diff
@@ -1919,3 +2012,14 @@ def _time_functions_patches():
         "google.cloud.storage.client.get_expiration_seconds_v4", return_value=10
     )
     return dtstamps_patch, now_patch, expire_secs_patch
+
+
+class _Bucket(object):
```

Comment:
Delete this class after refactoring above to use a mock rather than an actual `Blob`.